### PR TITLE
Delete unused paperclip columns.

### LINKED ISSUE #9726: Unused Paperclip columns still present
### What we should change and why (this is tech debt)

We replaced Paperclip with Active Storage but we kept the old Paperclip columns in the database tables for the transition. This transition is well over now and new devs get confused about inconsistent data. They look at the old Paperclip columns and wonder why they don't match up with uploaded files.

Removing the columns should also reduce the database size and make result sets smaller.

:warning: *This issue should be solved in two pull requests:*

1. Add the column to `ignored_columns` on each model. Otherwise we would encounter server errors during deployment because Rails would still try to access these columns when we just deleted them. This has to be deployed before the next pull request is merged.
2. Create database migrations to remove the columns. After they have been ignored, it's safe to delete the columns. The `ignored_columns` config should be removed as well now.

### Context

- #9126
- #9119

### PR DESCRIPTION
**Pull request by AthiraKadampatta**
#### What? Why?

Closes #9726

Removes unused paperclip columns from the following tables.
1. enterprises
2. enterprise_groups
3. terms_of_service_files
4. spree_assets

#### What should we test?

- Visit a shop page and verify that all images are still displayed (as much as before on staging).

#### Release notes

Technical changes

Remove unused paperclip columns from the following tables.
1. enterprises
2. enterprise_groups
3. terms_of_service_files
4. spree_assets

#### Dependencies

https://github.com/openfoodfoundation/openfoodnetwork/pull/9730

### CODE REVIEW

**Review by mkllnk — COMMENTED** [review]
Excellent. Let's wait for the other PR to be merge before proceeding with this. We can then rebase this PR and the tests should pass.

**Comment by AthiraKadampatta** [comment]
Hello @mkllnk I have rebased the PR and the tests are passing now. Please review.

**Review by mkllnk — APPROVED** [review]
Thank you! That's great. The other pull request has been merged and will be included in the next release on Friday. We'll be able to merge this one next week after testing.

**Comment by mkllnk** [comment]
Okay, previous PR has been deployed.

**Comment by drummer83** [comment]
Hi @AthiraKadampatta,
Thanks for finishing the second step of this issue. It was very interesting for me to test because I am new to PSQL and so I could learn a bit. :wink: 

**Before your PR**
I first made sure that the following things are in place and displayed correctly on the staging server:
- product images
- terms and conditions (of an enterprise)
- terms of service (of the instance)
- promo image and logo (of an enterprise)
- promo image and logo (of an enterprise group)

I also verified that I could see the respective columns in the psql tables:
```

                                                    Table "public.enterprises"
              Column               |            Type             | Collation | Nullable |                 Default                 
-----------------------------------+-----------------------------+-----------+----------+-----------------------------------------
 id                                | integer                     |           | not null | nextval('enterprises_id_seq'::regclass)
 name                              | character varying(255)      |           |          | 
 description                       | text                        |           |          | 
 long_description                  | text                        |           |          | 
 is_primary_producer               | boolean                     |           |          | 
 contact_name                      | character varying(255)      |           |          | 
 phone                             | character varying(255)      |           |          | 
 website                           | character varying(255)      |           |          | 
 twitter                           | character varying(255)      |           |          | 
 abn                               | character varying(255)      |           |          | 
 acn                               | character varying(255)      |           |          | 
 address_id                        | integer                     |           |          | 
 pickup_times                      | text                        |           |          | 
 next_collection_at                | character varying(255)      |           |          | 
 created_at                        | timestamp without time zone |           | not null | 
 updated_at                        | timestamp without time zone |           | not null | 
 distributor_info                  | text                        |           |          | 
 logo_file_name                    | character varying(255)      |           |          | 
 logo_content_type                 | character varying(255)      |           |          | 
 logo_file_size                    | integer                     |           |          | 
 logo_updated_at                   | timestamp without time zone |           |          | 
 promo_image_file_name             | character varying(255)      |           |          | 
 promo_image_content_type          | character varying(255)      |           |          | 
 promo_image_file_size             | integer                     |           |          | 
 promo_image_updated_at            | timestamp without time zone |           |          | 
 facebook                          | character varying(255)      |           |          | 
 instagram                         | character varying(255)      |           |          | 
 linkedin                          | character varying(255)      |           |          | 
 owner_id                          | integer                     |           | not null | 
 sells                             | character varying(255)      |           | not null | 'none'::character varying
 producer_profile_only             | boolean                     |           |          | false
 permalink                         | character varying(255)      |           | not null | 
 charges_sales_tax                 | boolean                     |           | not null | false
 email_address                     | character varying(255)      |           |          | 
 require_login                     | boolean                     |           | not null | false
 allow_guest_orders                | boolean                     |           | not null | true
 allow_order_changes               | boolean                     |           | not null | false
 invoice_text                      | text                        |           |          | 
 display_invoice_logo              | boolean                     |           |          | false
 enable_subscriptions              | boolean                     |           | not null | false
 terms_and_conditions_file_name    | character varying(255)      |           |          | 
 terms_and_conditions_content_type | character varying(255)      |           |          | 
 terms_and_conditions_file_size    | integer                     |           |          | 
 terms_and_conditions_updated_at   | timestamp without time zone |           |          | 
 business_address_id               | integer                     |           |          | 
 show_customer_names_to_suppliers  | boolean                     |           | not null | false
 visible                           | character varying(255)      |           | not null | 'public'::character varying
 whatsapp_phone                    | character varying(255)      |           |          | 

                                               Table "public.terms_of_service_files"
         Column          |            Type             | Collation | Nullable |                      Default                       
-------------------------+-----------------------------+-----------+----------+----------------------------------------------------
 id                      | integer                     |           | not null | nextval('terms_of_service_files_id_seq'::regclass)
 created_at              | timestamp without time zone |           | not null | 
 updated_at              | timestamp without time zone |           | not null | 
 attachment_file_name    | character varying           |           |          | 
 attachment_content_type | character varying           |           |          | 
 attachment_file_size    | integer                     |           |          | 
 attachment_updated_at   | timestamp without time zone |           |          | 

                                               Table "public.enterprise_groups"
          Column          |            Type             | Collation | Nullable |                    Default                    
--------------------------+-----------------------------+-----------+----------+-----------------------------------------------
 id                       | integer                     |           | not null | nextval('enterprise_groups_id_seq'::regclass)
 name                     | character varying(255)      |           |          | 
 on_front_page            | boolean                     |           |          | 
 position                 | integer                     |           |          | 
 promo_image_file_name    | character varying(255)      |           |          | 
 promo_image_content_type | character varying(255)      |           |          | 
 promo_image_file_size    | integer                     |           |          | 
 promo_image_updated_at   | timestamp without time zone |           |          | 
 description              | text                        |           |          | 
 long_description         | text                        |           |          | 
 logo_file_name           | character varying(255)      |           |          | 
 logo_content_type        | character varying(255)      |           |          | 
 logo_file_size           | integer                     |           |          | 
 logo_updated_at          | timestamp without time zone |           |          | 
 address_id               | integer                     |           |          | 
 email                    | character varying(255)      |           | not null | ''::character varying
 website                  | character varying(255)      |           | not null | ''::character varying
 facebook                 | character varying(255)      |           | not null | ''::character varying
 instagram                | character varying(255)      |           | not null | ''::character varying
 linkedin                 | character varying(255)      |           | not null | ''::character varying
 twitter                  | character varying(255)      |           | not null | ''::character varying
 owner_id                 | integer                     |           |          | 
 permalink                | character varying(255)      |           | not null | 

                                               Table "public.spree_assets"
         Column          |            Type             | Collation | Nullable |                 Default                  
-------------------------+-----------------------------+-----------+----------+------------------------------------------
 id                      | integer                     |           | not null | nextval('spree_assets_id_seq'::regclass)
 viewable_id             | integer                     |           |          | 
 attachment_width        | integer                     |           |          | 
 attachment_height       | integer                     |           |          | 
 attachment_file_size    | integer                     |           |          | 
 position                | integer                     |           |          | 
 viewable_type           | character varying(50)       |           |          | 
 attachment_content_type | character varying(255)      |           |          | 
 attachment_file_name    | character varying(255)      |           |          | 
 type                    | character varying(75)       |           |          | 
 attachment_updated_at   | timestamp without time zone |           |          | 
 alt                     | text                        |           |          | 
```

**After your PR**
I first checked that the following things are still displayed correctly on the staging server:
- product images :heavy_check_mark: 
- terms and conditions (of an enterprise) :heavy_check_mark: 
- terms of service (of the instance) :heavy_check_mark: 
- promo image and logo (of an enterprise) :heavy_check_mark: 
- promo image and logo (of an enterprise group) :heavy_check_mark: 

I also checked the database to see if the columns have been removed:

```
                                                   Table "public.enterprises"
              Column              |            Type             | Collation | Nullable |                 Default                 
----------------------------------+-----------------------------+-----------+----------+-----------------------------------------
 id                               | integer                     |           | not null | nextval('enterprises_id_seq'::regclass)
 name                             | character varying(255)      |           |          | 
 description                      | text                        |           |          | 
 long_description                 | text                        |           |          | 
 is_primary_producer              | boolean                     |           |          | 
 contact_name                     | character varying(255)      |           |          | 
 phone                            | character varying(255)      |           |          | 
 website                          | character varying(255)      |           |          | 
 twitter                          | character varying(255)      |           |          | 
 abn                              | character varying(255)      |           |          | 
 acn                              | character varying(255)      |           |          | 
 address_id                       | integer                     |           |          | 
 pickup_times                     | text                        |           |          | 
 next_collection_at               | character varying(255)      |           |          | 
 created_at                       | timestamp without time zone |           | not null | 
 updated_at                       | timestamp without time zone |           | not null | 
 distributor_info                 | text                        |           |          | 
 facebook                         | character varying(255)      |           |          | 
 instagram                        | character varying(255)      |           |          | 
 linkedin                         | character varying(255)      |           |          | 
 owner_id                         | integer                     |           | not null | 
 sells                            | character varying(255)      |           | not null | 'none'::character varying
 producer_profile_only            | boolean                     |           |          | false
 permalink                        | character varying(255)      |           | not null | 
 charges_sales_tax                | boolean                     |           | not null | false
 email_address                    | character varying(255)      |           |          | 
 require_login                    | boolean                     |           | not null | false
 allow_guest_orders               | boolean                     |           | not null | true
 allow_order_changes              | boolean                     |           | not null | false
 invoice_text                     | text                        |           |          | 
 display_invoice_logo             | boolean                     |           |          | false
 enable_subscriptions             | boolean                     |           | not null | false
 business_address_id              | integer                     |           |          | 
 show_customer_names_to_suppliers | boolean                     |           | not null | false
 visible                          | character varying(255)      |           | not null | 'public'::character varying
 whatsapp_phone                   | character varying(255)      |           |          | 

                                        Table "public.terms_of_service_files"
   Column   |            Type             | Collation | Nullable |                      Default                       
------------+-----------------------------+-----------+----------+----------------------------------------------------
 id         | integer                     |           | not null | nextval('terms_of_service_files_id_seq'::regclass)
 created_at | timestamp without time zone |           | not null | 
 updated_at | timestamp without time zone |           | not null | 

                                         Table "public.enterprise_groups"
      Column      |          Type          | Collation | Nullable |                    Default                    
------------------+------------------------+-----------+----------+-----------------------------------------------
 id               | integer                |           | not null | nextval('enterprise_groups_id_seq'::regclass)
 name             | character varying(255) |           |          | 
 on_front_page    | boolean                |           |          | 
 position         | integer                |           |          | 
 description      | text                   |           |          | 
 long_description | text                   |           |          | 
 address_id       | integer                |           |          | 
 email            | character varying(255) |           | not null | ''::character varying
 website          | character varying(255) |           | not null | ''::character varying
 facebook         | character varying(255) |           | not null | ''::character varying
 instagram        | character varying(255) |           | not null | ''::character varying
 linkedin         | character varying(255) |           | not null | ''::character varying
 twitter          | character varying(255) |           | not null | ''::character varying
 owner_id         | integer                |           |          | 
 permalink        | character varying(255) |           | not null | 

                                       Table "public.spree_assets"
    Column     |         Type          | Collation | Nullable |                 Default                  
---------------+-----------------------+-----------+----------+------------------------------------------
 id            | integer               |           | not null | nextval('spree_assets_id_seq'::regclass)
 viewable_id   | integer               |           |          | 
 position      | integer               |           |          | 
 viewable_type | character varying(50) |           |          | 
 type          | character varying(75) |           |          | 
 alt           | text                  |           |          | 

```

As far as I can tell this is looking really great! I will move this to 'Ready to Go'! :partying_face: 
Thanks again! :green_heart:

**Pull request merged**
